evap/student/templates/student_vote.html

```diff
@@ -200,6 +220,18 @@ <h3 class="mb-3">{{ evaluation.full_name }} ({{ evaluation.course.semester.name
                 bottomCard: document.querySelector("#bottom_text_results_publish_confirmation_card"),
             };
 
+            // Ensure that selected questionnaire language is saved and loaded
```

Comment:
when a language is stored in localstorage, the page always double-loads, even if the correct language is already shown.